apps/api/src/common/documentation.ts

```diff
@@ -0,0 +1,26 @@
+import { INestApplication } from '@nestjs/common';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { FormModule } from 'src/form/form.module';
+
+export const Documentation = (app: INestApplication) => {
```

Comment:
Why is this in common?

Reply:
Because it is referenced by other nestjs modules. Also, this is common for nestjs API and not for our monorepo 😄

Reply:
Ahh I misread that 👍